mava/components/building/best_checkpointer.py

```diff
@@ -6,15 +7,26 @@
 
 from mava.components.component import Component
 from mava.core_jax import SystemBuilder, SystemParameterServer
+from mava.utils.lp_utils import termination_fn
 
 
 @dataclass
 class BestCheckpointerConfig:
     checkpointing_metric: Tuple = ("mean_episode_return",)
     checkpoint_best_perf: bool = False
+    # Flag to calculate the absolute metric
+    absolute_metric: bool = False
+    # How many episodes to run evaluation for
+    absolute_metric_duration: Optional[Any] = None
 
 
 class BestCheckpointer(Component):
+    """A component that stores the best network parameters for two options:"""
+
+    """ 1- Checkpointing the best network for a specific metric.
+    2- Calculating the absolute metric.
+    """
+
```

Comment:
Can you make this 1 docstring?

Reply:
I tried but received an error related to mypy as it needs only one line for docstring

Reply:
I will try to find solution for it :raised_hands:

Reply:
Done!